src/main/java/com/fasterxml/jackson/datatype/joda/cfg/JacksonJodaDateFormat.java

```diff
@@ -242,7 +242,23 @@ public DateTimeFormatter rawFormatter() {
         return _formatter;
     }
 
+    /**
+     * @deprecated since 2.20 Use {@link #createFormatter(SerializerProvider, DateTimeZone)} instead
+     */
+    @Deprecated // since 2.20
```

Comment:
Ok, looks like there are many references to this now-deprecated method. I'll fix them.

But not 100% sure if this should be deprecated or not.

Reply:
Will remove deprecation... I should have reviewed this more carefully; will make some changes in a new PR.